images/detekt/pom.xml

```
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>sider</groupId>
+  <artifactId>runners</artifactId>
+  <version>1.0.0</version>
+  <dependencies>
+    <dependency>
+      <groupId>io.gitlab.arturbosch.detekt</groupId>
+      <artifactId>detekt-cli</artifactId>
+      <version>1.5.1</version>
```

Can you update it to the new version?

```suggestion
      <version>1.6.0</version>
```

- https://search.maven.org/artifact/io.gitlab.arturbosch.detekt/detekt-cli/1.6.0/jar
- https://github.com/arturbosch/detekt/releases/tag/1.6.0

---

The format of the HTML report was slightly changed, so handled it.